### PR TITLE
Libkms removal

### PR DESCRIPTION
**Pull request by FireBurn**
Libkms has been removed upstream, it turns out nothing was actually needing this, so updated the dependencies and use flags

https://gitlab.freedesktop.org/mesa/drm/-/commit/2b997bb4bb688be00620887c8646ff24ccb9396b

### CODE REVIEW

File: sys-boot/plymouth/plymouth-0.9.5-r1.ebuild [hunk 1]
```diff
@@ -72,7 +72,7 @@ src_configure() {
 		$(use_enable static-libs static)
 		$(use_enable debug tracing)
 		$(use_enable gtk gtk)
-		$(use_enable libkms drm)
+		$(use_enable drm drm)
```
Comment (mattst88) [inline sys-boot/plymouth/plymouth-0.9.5-r1.ebuild:75]: `$(use_enable drm)`

File: sys-boot/plymouth/plymouth-0.9.5-r1.ebuild [hunk 2]
```diff
@@ -21,7 +21,7 @@ HOMEPAGE="https://cgit.freedesktop.org/plymouth/"
 
 LICENSE="GPL-2"
 SLOT="0"
-IUSE="debug +gtk +libkms +pango +split-usr static-libs +udev"
+IUSE="debug +drm gtk +pango +split-usr static-libs +udev"
```
Comment (mattst88) [inline sys-boot/plymouth/plymouth-0.9.5-r1.ebuild:24]: You've dropped the `+` before `gtk`.

File: sys-boot/plymouth/plymouth-0.9.6_pre20211225.ebuild [hunk 3]
```diff
@@ -74,7 +74,7 @@ src_configure() {
 		$(use_enable static-libs static)
 		$(use_enable debug tracing)
 		$(use_enable gtk gtk)
-		$(use_enable libkms drm)
+		$(use_enable drm drm)
```
Comment (mattst88) [inline sys-boot/plymouth/plymouth-0.9.6_pre20211225.ebuild:77]: `$(use_enable drm)`

File: sys-boot/plymouth/plymouth-22.02.122.ebuild [hunk 4]
```diff
@@ -72,7 +72,7 @@ src_configure() {
 		$(use_enable static-libs static)
 		$(use_enable debug tracing)
 		$(use_enable gtk gtk)
-		$(use_enable libkms drm)
+		$(use_enable drm drm)
```
Comment (mattst88) [inline sys-boot/plymouth/plymouth-22.02.122.ebuild:75]: `$(use_enable drm)`

File: sys-boot/plymouth/plymouth-9999.ebuild [hunk 5]
```diff
@@ -72,7 +72,7 @@ src_configure() {
 		$(use_enable static-libs static)
 		$(use_enable debug tracing)
 		$(use_enable gtk gtk)
-		$(use_enable libkms drm)
+		$(use_enable drm drm)
```
Comment (mattst88) [inline sys-boot/plymouth/plymouth-9999.ebuild:75]: `$(use_enable drm)`

**Review by mattst88 — CHANGES_REQUESTED** [review]
We need to revbump the modified plymouth and xf86-video-vmware ebuilds.

File: sys-boot/plymouth/plymouth-0.9.5-r1.ebuild (hunk 1 above)
Comment (FireBurn) [reply sys-boot/plymouth/plymouth-0.9.5-r1.ebuild]: I've fixed that, reordered and also sorted the double gtk gtk too

File: sys-boot/plymouth/plymouth-0.9.5-r1.ebuild (hunk 2 above)
Comment (FireBurn) [reply sys-boot/plymouth/plymouth-0.9.5-r1.ebuild]: Whoops, good spot, sorted now

**Comment by mattst88** [comment]
Thanks!

**Pull request closed without merging**